requirements/pip.txt

```diff
@@ -64,7 +64,7 @@ git+https://github.com/elastic/elasticsearch-dsl-py@97e3f756a8cacd1c863d3ced3d17
 # ImportError: cannot import name 'Document'
 django-elasticsearch-dsl==0.5.1  # pyup: ignore
 selectolax==0.2.1
-orjson==2.0.7
+orjson==2.1.0
```

Comment:
> Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-ilw45mqk/orjson/

That's from travis

Reply:
I ignored this upgrade and triggered a new test on travis. Let's see if it pass. We can investigate later how to fix this if needed.